diff -up corekeyboard-v4.0.0/trayicon.h.omv~ corekeyboard-v4.0.0/trayicon.h
--- corekeyboard-v4.0.0/trayicon.h.omv~	2020-11-27 21:06:22.067104642 +0100
+++ corekeyboard-v4.0.0/trayicon.h	2020-11-27 21:06:33.741220326 +0100
@@ -65,7 +65,7 @@ class trayicon : public QSystemTrayIcon
 		void activationHandler( QSystemTrayIcon::ActivationReason reason ) {
 
 			if ( reason == QSystemTrayIcon::Trigger ) {
-				if ( settingsManage::initialize( true )->value( "CoreApps", "TouchMode" ) == true )
+				if ( settingsManage::initialize( true )->value( "CoreApps", "TouchMode" ) )
 					contextMenu()->exec( QCursor::pos() );
 
 				else
